packages/shared/src/components/feeds/FeedSettingsButton.tsx

```diff
@@ -0,0 +1,33 @@
+import React, { ReactElement } from 'react';
+import {
+  Button,
+  ButtonProps,
+  ButtonSize,
+  ButtonVariant,
+} from '../buttons/Button';
+import { useAnalyticsContext } from '../../contexts/AnalyticsContext';
+import { AnalyticsEvent } from '../../lib/analytics';
+import { LazyModal } from '../modals/common/types';
+import { useLazyModal } from '../../hooks/useLazyModal';
+import { FilterIcon } from '../icons';
+
+export function FeedSettingsButton(props: ButtonProps<'button'>): ReactElement {
```

Comment:
I guess we don't use this for the feed settings button on feed? should we?

Reply:
Yeah, it is probably good to use this on that component too. Will update this in a moment.